screenpy_requests/questions/headers_of_the_last_response.py

```diff
@@ -1,15 +1,17 @@
-"""
-Investigate the headers of the last API response received.
-"""
+"""Investigate the headers of the last API response received."""
 
-from typing import MutableMapping
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, MutableMapping
```

Comment:
We can move the `MutableMapping` into the `if TYPE_CHECKING` block.

Reply:
Same deal here as the Union.  I'm guessing ruff sees this as a no-op move so it opts to leave it alone.